source/Xunit.Gherkin.Quick/CoreModel/Feature/FeatureClassInfo.cs

```diff
@@ -1,17 +1,28 @@
 ﻿using System;
 using System.Reflection;
+using System.Text.RegularExpressions;
 
 namespace Xunit.Gherkin.Quick
 {
     internal sealed class FeatureClassInfo
     {
         public string FeatureFilePath { get; }
+        private Regex _matcher;
+        public bool IsPattern { get { return FeatureFilePath.Contains("*"); } }

```

Comment:
This getter is too expensive - string operation is more expensive than null check. So here, `return _matcher !== null` is better.

Reply:
Done!